index.html

```diff
@@ -151,12 +152,14 @@ <h1>Contact me</h1>
     <p class="second-text">If you have an application you are interested in developing, a feature that you need built
       or a project that needs coding.
       I’d love to help with it!</p>
-    <div class="form">
-      <input type="text" placeholder="Lorem Ipsum">
-      <input type="text" placeholder="Loremipsum@gmail.com">
-      <textarea placeholder="Write your message here..." rows="10"></textarea>
-      <button>Get in touch</button>
-    </div>
+    <form class="form" action="https://formspree.io/f/meqwlynw" method="post">
+      <input type="text" placeholder="Lorem Ipsum" required maxlength="30">
```

Comment:
- [ ] In order to send data when the form is submitted this input should have a `name` attribute, please refer to [name Attribute](https://www.w3schools.com/tags/att_input_name.asphttps://www.w3schools.com/tags/att_input_name.asp) for more information 👍🏼 
- [ ] Ensure that this input has a meaningful placeholder such as `Full Name`, or `Enter your name here!` ...etc.
- [ ] Also Ensure that this input is required input so when the user tries to submit the form he will be notified if this input is empty 👌🏼